apps/prairielearn/src/pages/partials/question.ejs

```diff
@@ -107,7 +107,7 @@
         <% } %>
         <% // Show even when question_copy_targets is empty. %>
         <% // We'll show a CTA to request a course if the user isn't an editor of any course. %>
-        <% if (course.template_course && question_context != 'manual_grading') { %>
+        <% if (typeof question_copy_targets !== 'undefined' && course.template_course && question_context != 'manual_grading') { %>
```

Comment:
This change ensures that the "Copy question" button isn't shown on pages where we haven't called `setQuestionCopyTargets(res)`. This `typeof ...` is the same conditional used to decide to render the copy modal or not.